app/subscriber/src/features/my-folders/MyFolders.tsx

```diff
@@ -40,7 +40,7 @@ export const MyFolders = () => {
 
   const handleSave = () => {
     if (!!active) {
-      updateFolder(active).then(() => {
+      updateFolder(active).then((data) => {
```

Comment:
`data` is never used.  `active` will not have correct `version`.  Edit this folder twice in a row will result in a failure.